server/pulp/plugins/loader/api.py

```diff
@@ -432,15 +432,16 @@ def load_content_types(types_dir=_TYPES_DIR, dry_run=False, drop_indices=False):
 
     # to handle node.json only
     descriptors = _load_type_descriptors(types_dir)
-    definitions = parser.parse(descriptors)
+    old_definitions = parser.parse(descriptors)
 
     # get information about content unit types from entry points
-    definitions += _generate_plugin_definitions()
+    new_definitions = _generate_plugin_definitions()
```

Comment:
let's make this variable name better.   s/new_definitions/mongoengine_definitions/